variables.json.example

```diff
@@ -0,0 +1,11 @@
+{
+  "server_hostname": "foreman.example.com",
```

Comment:
:-1: on this. If needed, it can be parsed from the base_url.

Reply:
See https://github.com/theforeman/smoker/pull/11#discussion_r539310678